config/prometheus/prometheus-server-service.yaml

```diff
@@ -0,0 +1,35 @@
+---
+# Source: prometheus/templates/services/server.yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: prometheus
+  name: prometheus-server
```

Comment:
Can we just name this `prometheus` so when we have URL support we can have a default of `http://prometheus:9090` ?

Reply:
Want to avoid `prometheus` since that has a potential for conflict. Will use `rso-prometheus` below and remove this one.

Reply:
along that vein I should probably change all the basic 'prometheus' names to something more specific. ugh.

Reply:
I like the change of the name prefix for all the other objects, I still don't like it on the service because of the whole sensible defaults thing.

A default of "http://prometheus:9090" is a good default because it is likely to actually hit (our Prometheus or theirs, it's generic); if the service name is "redsky-prometheus" then we either need to make the default URL "http://redsky-prometheus:9090" (only likely to hit our Prometheus) or stick with "http://prometheus:9090" and have a default that needs to be overwritten in the particular use case we established the default behavior for in the first place.

I'm fine to revisit this once we are closer with URL based metric collection.